stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/TypeAdapterGenerator.java

```diff
@@ -831,5 +945,9 @@ void addField(@NotNull TypeMirror fieldType, @NotNull String fieldName) {
         void addTypeToAdapterAccessor(@NotNull TypeMirror typeMirror, String accessorCode) {
             mAdapterAccessor.put(typeMirror.toString(), accessorCode);
         }
+
+        void addFieldToAccessor(@NotNull String  fieldName, String accessorCode){
```

Comment:
`accessorCode` should also be annotated either NotNull or Nullable depending on what is expected.